### ISSUE TITLE
Bug: unable to select some packages

### ISSUE THREAD

**Issue by ppvnf** [martinet101/WingetUI#238]
I am unable to select the 6th package (regular size) and the 11th package (when maximized):

Examples:

7-Zip Alpha (exe) in the first image (regular window)
![Captura de tela 2022-10-12 133253](https://user-images.githubusercontent.com/73800734/195399176-e72015ed-47ea-40da-a44e-8f42d3381943.png)

7ztm in the second one (maximized window)
![Captura de tela 2022-10-12 133316](https://user-images.githubusercontent.com/73800734/195399183-3c9973a4-efc3-46af-9d5f-9dee80defc52.png)

I can confirm it affects both Install and Uninstall sections. I didn't test it on Update section. Also, when I scroll the list  the affected package changes

**Comment by martinet101** [comment]
Yes, this issue has been already fixed. It was a label showing over the tree widget.
